test/simplify_algebra_test.cpp

```diff
@@ -734,6 +734,25 @@ TEST_CASE(simplify_div_const)
     EXPECT(m1 == m2);
 }
 
+TEST_CASE(simplify_unit_div_const)
+{
+    migraphx::module m1;
+    {
+        auto x    = m1.add_parameter("x", {migraphx::shape::int32_type, {1}});
+        auto unit = m1.add_literal(1);
+        m1.add_instruction(migraphx::make_op("div"), x, unit);
```

Comment:
I think you can add a return or just add in an extra operator so it doesnt get replaced with identity:

```cpp
        auto unit = m1.add_literal(1);
        auto div = m1.add_instruction(migraphx::make_op("div"), x, unit);
        m1.add_instruction(migraphx::make_op("relu"), div);
```

```cpp
        auto unit = m1.add_literal(1);
        auto div = m1.add_instruction(migraphx::make_op("div"), x, unit);
        m1.add_return({div});
```